frontend/src/hooks/useClearSWRCache.test.ts

```diff
@@ -23,8 +23,19 @@ describe('manageCacheEntries', () => {
 
         clearCacheEntries(cacheMock, 'prefix-4', 'prefix-', 2);
 
-        expect(cacheMock.has('prefix-4')).toBe(true);
-        expect([...cacheMock.keys()].length).toBe(2);
+        expect([...cacheMock.keys()]).toStrictEqual(['prefix-3', 'prefix-4']);
+    });
+
+    it('should not delete the current key, even if it would be cleared as part of the cache size limit', () => {
+        const cacheMock = new Map();
+        cacheMock.set('prefix-1', {});
+        cacheMock.set('prefix-2', {});
+        cacheMock.set('prefix-3', {});
+        cacheMock.set('prefix-4', {});
+
+        clearCacheEntries(cacheMock, 'prefix-2', 'prefix-', 2);
```

Comment:
wondering if we should also clean the signature of clearCacheEntries. With 4 params it's totally unclear what each of those means.